test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -975,6 +975,32 @@ def forward(self, x):
                 v['node_output']['mq'][0]['ref_node_target_type'],
                 v['node_output']['mq'][0]['sqnr']])
 
+    def test_needs_dtype_transform_on_outputs(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = nn.Sequential(
+                    nn.Sequential(
+                        torch.nn.Conv2d(1, 1, 1),
+                    )
+                )
+
+            def forward(self, x):
+                x1 = self.conv(x)
+                return x1
+
+        m = M().eval()
+        m.qconfig = torch.quantization.default_qconfig
+        example_args = (torch.randn(1, 1, 2, 2),)
+        mp = _quantize_dbr.prepare(m, example_args)
+        self.assertTrue(
```

Comment:
can we add some comment to explain what this test means?

Reply:
sure.  I'm not sure this code will need to land, will depend on if a future iteration of https://github.com/pytorch/pytorch/pull/69002 can be a performance win.  If it works out, we can improve the tests before going through review.

for now I'm just saving this work so I can resume it later.